ios/FluentUI/HUD/HeadsUpDisplayTokenSet.swift

```diff
@@ -47,13 +36,22 @@ public class HeadsUpDisplayTokenSet: ControlTokenSet<HeadsUpDisplayTokenSet.Toke
                 }
 
             case .activityIndicatorColor:
+                let colorToken = theme.aliasTokens.colors[.foregroundLightStatic]
```

Comment:
Not sure it's quite in the scope of this one, but the design also seems to have a "Progress indicator background" value defined and the code doesn't.

Reply:
Some other controls like cards have different designs as well, and for the scope of the colors work the focus has been on updating the colors, fonts, sizes, etc.